src/md-collection-loader/description.js

```diff
@@ -0,0 +1,39 @@
+import remark from "remark"
+import stripMd from "strip-markdown"
+import { prune }  from "underscore.string"
+
+const defaultOpts = {
+  pruneLength: 140,
+  pruneString: "…",
```

Comment:
pruneString was not count into length
Test: https://github.com/epeli/underscore.string/blob/master/tests/prune.js

"Limited to X chars"

prune 6 '...'

"Limited..." (always get the hold word)